providers/aws/resources/aws.lr

```diff
@@ -1822,17 +1822,19 @@ private aws.s3.bucket.corsrule @defaults("name") {
 }
 
 // Amazon S3 bucket policy
-private aws.s3.bucket.policy @defaults("name version") {
+private aws.s3.bucket.policy @defaults("bucketName exists version") {
   // Unique ID for the policy
   id string
-  // Name for the policy
-  name string
+  // Bucket name that this policy belongs
+  bucketName string
```

Comment:
This might be a breaking change, but a bucket policy doesn't have a name. It belongs to a bucket and, to calculate the bucket policy ID we need the bucket name, so I re purpose this field.

If this is not acceptable, I am ok creating an internal struct to pass the bucket name but, we need to do something about this `name`field that says it is the name of the policy, but it's not.

Reply:
we don't know whether this is used by someone or not. Perhaps keep both for a while and add a comment to `name` that it's deprecated. We can remove it in next major release